packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -264,6 +271,11 @@ const EAPostsItem = ({
   } = usePostsItem(props);
   const {onClick} = useClickableCell({href: postLink});
   const cardView = viewType === "card";
+  const {postContents} = usePostContents({
+    post,
+    fragmentName: "PostsList",
+    skip: !cardView,
+  });
```

Comment:
Why do we need this? This seems potentially expensive and we should already have the post list data.

Reply:
Could we skip if it's not a crosspost?

Reply:
That's the point of `usePostContents` - it only makes a request if it's a foreign crosspost, otherwise it just returns the fields from the object that's passed in.